llvm/test/tools/llvm-mca/RISCV/MIPS/p8700.s

```diff
@@ -0,0 +1,143 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
+# RUN: llvm-mca -mtriple=riscv64 -mcpu=mips-p8700 -timeline -iterations=1 < %s | FileCheck %s
+
+# A few instructions to test the pipeline:
+# - Integer division (IDiv) exercises the p8700GpDiv resource.
+# - Integer multiplication (IMul) uses p8700GpMul.
+# - Floating-point multiplication uses the FPUL pipeline.
+# - Load/Store instructions use the LSU pipeline.
+# - Simple ALU instructions test the p8700WriteEitherALU and p8700IssueAL2 resources.
+# - A jump instruction to test the CTI pipeline.
+
+  .text
+  .globl _start
+_start:
```

Comment:
Actually you don't need this in a MCA test. :-)

Reply:
Nice catch! Thank you!

The tests in the LLVM project keep growing larger, and while we usually trim them manually or use `llvm-reduce`, it would be very helpful to have a dedicated tool or utility that can automatically remove unnecessary parts from tests in general.